js/libs/ui-shared/src/controls/PasswordInput.tsx

```diff
@@ -6,9 +6,9 @@ import { EyeIcon, EyeSlashIcon } from "@patternfly/react-icons";
 import {
   KeycloakTextInput,
   KeycloakTextInputProps,
-} from "../keycloak-text-input/KeycloakTextInput";
+} from "../../../../apps/admin-ui/src/components/keycloak-text-input/KeycloakTextInput";
```

Comment:
This should probably import `KeycloakTextInput` from `ui-shared` instead. Perhaps we remove `KeycloakTextInput` from the `admin-ui` entirely and replace it with the one from `ui-shared`?

Reply:
as `KeycloakTextInput` is going to go away anyway we can leave this for now